Jenkinsfile

```diff
@@ -298,6 +298,19 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
             }
         }
 
+
+        if (!params.MINIMAL) {
+            stage('Kola Runs') {
+                container('jnlp') {
+                    utils.shwrap("""
+                        oc start-build --wait fedora-coreos-pipeline-kola-aws \
```

Comment:
Suggestion: let's not `--wait` for this? Essentially, consider kola AWS testing as a follow-up job, not something that's necessarily part of the build process itself.

Reply:
hmm interesting - i would have figured we'd wanted it that way.

Reply:
One of the reasons why I suggested this is that kola AWS tests can take a while, and there's no reason to delay building other things while that's ongoing (though related: https://github.com/coreos/fedora-coreos-pipeline/issues/68). I also think errorring the build job if the AWS test fails isn't practical.